### ISSUE TITLE
Upgraded to scribe 3, ReflectionException <controller> does not exist

### ISSUE THREAD

**Issue by dwdozier** [knuckleswtf/scribe#411]
- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.knuckles.wtf/laravel/troubleshooting).**

**What happened?**
1. I set my configuration to... - already set
2. Then I ran `php artisan scribe:upgrade` 
2. Then I ran `php artisan scribe:generate` ...
3. But I saw...
```sh
   ReflectionException

  Class App\Api\V3\Controllers\App\Api\V3\Controllers\MeetingsController does not exist
```

My problem is I can't determine why my path is incorrect, that controller exists at `App\Api\V3\Controllers\MeetingsController ` but it is being prepended with `App\Api\V3\Controllers\`? From what I can tell I need to remove the `uses` from the routes file it works fine

Commented out and successful run.
```sh
//use App\Api\V3\Controllers\MeetingsController;
//use App\Api\V3\Controllers\UsersController;
```

Apologies if I missed something obvious.

instead of...
standard processing messages

**Screenshots and stack traces:**

**My environment:**
 - PHP version (from `php -v`): 7.4
 - Framework (Laravel/Lumen): Laravel
 - Laravel/Lumen version (from `composer show laravel/framework` or `composer show laravel/lumen-framework`): 8.8
 - Scribe version (from `composer show knuckleswtf/scribe`): 3.3.1

**My Scribe config (minus the comments):**
```sh
<?php

use Knuckles\Scribe\Extracting\Strategies;

return [
    'theme' => 'default',
    'title' => null,
    'description' => '',
    'base_url' => null,
    'routes' => [[
        'match' => [
            'domains' => ['*'],
            'prefixes' => ['api/v3/*'],
            'versions' => ['v1'],
        ],
        'include' => [],
        'exclude' => [],
        'apply' => [
            'headers' => ['Content-Type' => 'application/json', 'Accept' => 'application/json'],
            'response_calls' => [
                'methods' => [],
                'config' => ['app.env' => 'documentation'],
                'queryParams' => [],
                'bodyParams' => [],
                'fileParams' => [],
                'cookies' => [],
            ],
        ],
    ]],
    'type' => 'laravel',
    'static' => [
        'output_path' => 'public/docs',
    ],
    'laravel' => [
        'add_routes' => true,
        'docs_url' => '/docs',
        'middleware' => [],
    ],
    'try_it_out' => [
        'enabled' => true,
        'base_url' => null,
    ],
    'auth' => [
        /*
         * Set this to true if any endpoints in your API use authentication.
         */
        'enabled' => true,
        'default' => true,
        'in' => 'bearer',
        'name' => 'hahaha',
        'use_value' => env('SCRIBE_AUTH_KEY'),
        'placeholder' => '{ACCESS_TOKEN}',
        'extra_info' => 'You can retrieve your token by visiting your dashboard and clicking <b>Generate API token</b>.',
    ],
    'intro_text' => <<<INTRO
This documentation aims to provide all the information you need to work with our API.

<aside>As you scroll, you'll see code examples for working with the API in different programming languages in the dark area to the right (or as part of the content on mobile).
You can switch the language used with the tabs at the top right (or from the nav menu at the top left on mobile).</aside>
INTRO

    ,
    'example_languages' => [
        'bash',
        'javascript',
        'php',
        'python',
    ],
    'postman' => [
        'enabled' => true,
        'overrides' => [
            // 'info.version' => '2.0.0',
        ],
    ],
    'openapi' => [
        'enabled' => true,
        'overrides' => [
            // 'info.version' => '2.0.0',
        ],
    ],
    'default_group' => 'Endpoints',
    'logo' => 'img/logo_email.png',
    'faker_seed' => null,
    'strategies' => [
        'metadata' => [
            Strategies\Metadata\GetFromDocBlocks::class,
        ],
        'urlParameters' => [
            Strategies\UrlParameters\GetFromLaravelAPI::class,
            Strategies\UrlParameters\GetFromLumenAPI::class,
            Strategies\UrlParameters\GetFromUrlParamTag::class,
        ],
        'queryParameters' => [
            Strategies\QueryParameters\GetFromFormRequest::class,
            Strategies\QueryParameters\GetFromInlineValidator::class,
            Strategies\QueryParameters\GetFromQueryParamTag::class,
        ],
        'headers' => [
            Strategies\Headers\GetFromRouteRules::class,
            Strategies\Headers\GetFromHeaderTag::class,
        ],
        'bodyParameters' => [
            Strategies\BodyParameters\GetFromFormRequest::class,
            Strategies\BodyParameters\GetFromInlineValidator::class,
            Strategies\BodyParameters\GetFromBodyParamTag::class,
        ],
        'responses' => [
            Strategies\Responses\UseTransformerTags::class,
            Strategies\Responses\UseResponseTag::class,
            Strategies\Responses\UseResponseFileTag::class,
            Strategies\Responses\UseApiResourceTags::class,
            Strategies\Responses\ResponseCalls::class,
        ],
        'responseFields' => [
            Strategies\ResponseFields\GetFromResponseFieldTag::class,
        ],
    ],

    'fractal' => [
        'serializer' => null,
    ],
    'routeMatcher' => \Knuckles\Scribe\Matching\RouteMatcher::class,
    'database_connections_to_transact' => [config('database.default'), config('database.default')]
];

```
**Additional info:**

**Comment by shalvah** [comment]
Hmm, that is indeed strange. That path is likely provided to Scribe by a Laravel API. But this information isn't sufficient; we need the stack trace to know where.

**Comment by dwdozier** [comment]
```sh
[2022-01-20 19:03:13] local.ERROR: Class App\Api\V3\Controllers\App\Api\V3\Controllers\MeetingsController does not exist {"exception":"[object] (ReflectionException(code: -1): Class App\\Api\\V3\\Controllers\\App\\Api\\V3\\Controllers\\MeetingsController does not exist at /var/www/app.boardable.com/vendor/knuckleswtf/scribe/src/Commands/GenerateDocumentation.php:200)
[stacktrace]
#0 /var/www/app.boardable.com/vendor/knuckleswtf/scribe/src/Commands/GenerateDocumentation.php(200): ReflectionClass->__construct()
#1 /var/www/app.boardable.com/vendor/knuckleswtf/scribe/src/Commands/GenerateDocumentation.php(99): Knuckles\\Scribe\\Commands\\GenerateDocumentation->doesControllerMethodExist()
#2 /var/www/app.boardable.com/vendor/knuckleswtf/scribe/src/Commands/GenerateDocumentation.php(322): Knuckles\\Scribe\\Commands\\GenerateDocumentation->extractEndpointsInfoFromLaravelApp()
#3 /var/www/app.boardable.com/vendor/knuckleswtf/scribe/src/Commands/GenerateDocumentation.php(56): Knuckles\\Scribe\\Commands\\GenerateDocumentation->extractEndpointsInfoAndWriteToDisk()
#4 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Knuckles\\Scribe\\Commands\\GenerateDocumentation->handle()
#5 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Container/Util.php(40): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#6 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#7 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#8 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Container/Container.php(653): Illuminate\\Container\\BoundMethod::call()
#9 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Illuminate\\Container\\Container->call()
#10 /var/www/app.boardable.com/vendor/symfony/console/Command/Command.php(298): Illuminate\\Console\\Command->execute()
#11 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Console/Command.php(121): Symfony\\Component\\Console\\Command\\Command->run()
#12 /var/www/app.boardable.com/vendor/symfony/console/Application.php(1005): Illuminate\\Console\\Command->run()
#13 /var/www/app.boardable.com/vendor/symfony/console/Application.php(299): Symfony\\Component\\Console\\Application->doRunCommand()
#14 /var/www/app.boardable.com/vendor/symfony/console/Application.php(171): Symfony\\Component\\Console\\Application->doRun()
#15 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Console/Application.php(94): Symfony\\Component\\Console\\Application->run()
#16 /var/www/app.boardable.com/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(129): Illuminate\\Console\\Application->run()
#17 /var/www/app.boardable.com/artisan(35): Illuminate\\Foundation\\Console\\Kernel->handle()
#18 {main}
"}
[2022-01-20 19:03:13] local.DEBUG: Class App\Api\V3\Controllers\App\Api\V3\Controllers\MeetingsController does not exist
```

**Comment by dwdozier** [comment]
I should have included the route info as well:

```sh
use App\Api\V3\Controllers\MeetingsController;
//use App\Api\V3\Controllers\UsersController;

Route::resource('meetings', MeetingsController::class)->except([
    'create', 'edit'
]);
```

**Comment by dwdozier** [comment]
It appears this issue can be brought about by mixing the old and newer laravel route syntax. We had the above. for a couple of routes, then on others we had:

```sh
Route::get('me', 'MiscController@me');
```

which probably triggered whatever to prepend the `uses` twice.

Not to jump scopes but I had noticed that the v3.3.1 version we had installed wasn't the latest, so on a whim (ugh) I tried updating to the latest (v3.21.0) just in case, but once we resolved the above issue, I could no longer view the endpoints in the nav (something regarding Jets?). I was eventually able to find a release (v.3.16.0) that didn't encounter the issue and everything loaded as expected. We use the tryItOut feature and when I tested it against our product I wound up with:

![image](https://user-images.githubusercontent.com/2555178/150611874-f9e91b5e-4272-4d41-a1ac-8b1d86a06275.png)

So, unfortunately, I had to revert to the latest v2 of scribe which got me back to where I needed to be. My guess is that we are loading the same frontend libraries from a CDN and they are blocking each other maybe? I won't have time to come back to this right away, but I will definitely give you a better synopsis of what I am encountering. We are potentially moving away from loading client-side libraries from a CDN, so when that happens I may try again.
